Grammar/python.gram

```diff
@@ -1375,7 +1375,9 @@ invalid_case_block:
         RAISE_INDENTATION_ERROR("expected an indented block after 'case' statement on line %d", a->lineno) }
 invalid_as_pattern:
     | or_pattern 'as' a="_" { RAISE_SYNTAX_ERROR_KNOWN_LOCATION(a, "cannot use '_' as a target") }
-    | or_pattern 'as' !NAME a=expression { RAISE_SYNTAX_ERROR_KNOWN_LOCATION(a, "invalid pattern target") }
+    | or_pattern 'as' a=expression {
+        RAISE_SYNTAX_ERROR_KNOWN_LOCATION(
+            a, "cannot use pattern target as %s", _PyPegen_get_expr_name(a)) }
```

Comment:
The error message reads strange to me. Shouldn't it rather be `cannot use %s as pattern target`?

This would also make it analogous to the case before, `cannot use '_' as a target`

Reply:
Thanks, good idea, changing!